docs/en/uninstall.md

```diff
@@ -0,0 +1,38 @@
+To uninstall the **AI Video Super Resolution** solution, you must delete the AWS CloudFormation stack. This will delete all the resources created by the template except the S3 buckets starting with `superresolutionstack-superresolutionstorage` and `superresolutionstack-superresolutionbucketaccessl`. These two buckets will be retained when the solution stack is deleted in order to help prevent accidental data loss. 
```

Comment:
if i remember correctly, we should delete s3 first, otherwise the whole cloudformation will shown delete failed for the first time since s3 can only be delete explicitly